src/auth/respose/generate-token.respose.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class GenerateTokenRespose {
```

Comment:
typo: `Response` instead of `Respose` (also the file name)

Reply:
Thanks for review that error. Revised on other files with the same problem.
[refactor: revised that Response instead of Respose](https://github.com/Cophr/backend/pull/10/commits/4dbb6e9c12521add4584b5a6e6fe20e95aa4e8a3)